src/organizations/OrganizationEntity.js

```diff
@@ -40,6 +46,24 @@ export class OrganizationEntity extends Entity {
 		return this._entity && this._entity.properties && this._entity.properties.endDate;
 	}
 
+	get isCompletionTracked() {
```

Comment:
Although this makes sense as a `get` Does other entities do this now? If not I'd leave it as a method so people have the same experience.